src/domain/accessibilityStatement/AccessibilityStatement.tsx

```diff
@@ -0,0 +1,519 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+import 'swagger-ui-react/swagger-ui.css';
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+
+import Breadcrumb from '../../common/components/breadcrumb/Breadcrumb';
+import ExternalLink from '../../common/components/externalLink/ExternalLink';
+import { ROUTES, SUPPORT_EMAIL } from '../../constants';
+import useLocale from '../../hooks/useLocale';
+import { Language } from '../../types';
+import Container from '../app/layout/container/Container';
+import MainContent from '../app/layout/mainContent/MainContent';
+import PageWrapper from '../app/layout/pageWrapper/PageWrapper';
+import styles from './accessibilityStatement.module.scss';
+
+const AccessibilityStatementPage: React.FC = () => {
+  const locale = useLocale();
+  const { t } = useTranslation();
+  const accessibilityRequirementsWebsiteUrl =
+    'https://www.saavutettavuusvaatimukset.fi';
+  const accessibilityRequirementsWebsiteUrlSv =
+    'https://www.tillganglighetskrav.fi';
+
+  const getContent = (locale: Language) => {
+    switch (locale) {
+      case 'en':
+        return (
+          <>
+            <h1>Accessibility of Linked Events -web service</h1>
+            <p>
+              This accessibility statement explains how well the Linked Events
+              web service complies with the Act on the Provision of Digital
+              Services, what shortcomings there are in the accessibility of the
+              online service and how you can give us feedback on accessibility
+              problems.
+            </p>
+            <p>
+              The city of Helsinki is responsible for this online service. We
+              want as many users as possible to be able to access our digital
+              services. We take accessibility into account in the development of
+              digital services.
+            </p>
+
+            <h2>How accessible is the online service?</h2>
+            <p>
+              This online service partly meets the A and AA level accessibility
+              criteria required by law (WCAG criteria 2.1). The online service
+              has some shortcomings in accessibility, which are described in
+              more detail below.
+            </p>
+
+            <h2>Did you notice any gaps in accessibility?</h2>
+            <p>
+              We are constantly striving to improve the accessibility of the
+              online service. If you observe any issues that are not described
+              on this page, please let us know and we will do our best to fix
+              any shortcomings. We will respond to requests and comments as soon
+              as possible, but no later than within two weeks.
+            </p>
+            <p>
+              You can contact us by sending an email to{' '}
+              <a href={`mailto:${SUPPORT_EMAIL}`}>${SUPPORT_EMAIL}</a> / via an{' '}
```

Comment:
Extra $ sign:
<img width="217" alt="Screenshot 2023-12-01 at 11 13 12" src="https://github.com/City-of-Helsinki/linkedcomponents-ui/assets/66477579/900df4e0-e028-49e6-9150-2fca8f930810">

Reply:
Good catch. Removed the extra $.